tms_objects2.erb

```diff
@@ -319,6 +319,15 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
               <lido:term><%= l["Culture"] %></lido:term> 
             </lido:culture> 
         <% end %>   <%# Objculture |l| %>
+        <% if Dated != '' %> 
```

Comment:
unless Dated.nil?, unless DateBegin.nil? etc rather than ifs, and get example objects without these fields to test logic.